FEED_CHANGELOG.md

```diff
@@ -0,0 +1,2388 @@
+# Changes to NI Linux RT Package Feeds
+
+## Description
+
+This documents the full set of changes to individual packages in the core/ and extra/ feeds for each version release.
+
+The changes are too long to keep in the [Changelog](/CHANGELOG.md) itself, so they are kept here
+
+## Changes from Version 8.11 to Version 9.0
+
+### Changes to core/
```

Comment:
I think this is probably a lot more info than users are looking for in a markdown file. I think they would prefer to have a more condensed digest of "what was added/removed". Something like...

```md
## Feed Changes from Version 8.11 to Version 9.0

### core/
* Added
    * mypaint-brushes-1.0
    * mypaint-brushes-1.0-dbg
    * mypaint-brushes-1.0-dev
* Removed
    * xorg-minimal-fonts
```

* Disregard package version bumps. There will be too many of them to be comprehensible.
* Disregard packages with the `-lic` suffix. They are just license subpackages, and readers probably won't care about them in the context of this document.
* Keep each entry as concise as possible.
* Remove information about sizes and dependencies changing. That data is useful, but not for this digested document, I don't think.
    * If we decide to follow my suggestion about splitting up the scripts that generate this file, and keeping a json log of all the raw *change* data, then future users can pull this more-detailed information from the json, if they are interested.